inspirehep/modules/records/receivers.py

```diff
@@ -178,12 +178,35 @@ def enhance_after_index(sender, json, *args, **kwargs):
     populate_affiliation_suggest(sender, json, *args, **kwargs)
     populate_author_count(sender, json, *args, **kwargs)
     populate_authors_full_name_unicode_normalized(sender, json, *args, **kwargs)
+    populate_author_suggest(sender, json, *args, **kwargs)
     populate_earliest_date(sender, json, *args, **kwargs)
     populate_inspire_document_type(sender, json, *args, **kwargs)
     populate_name_variations(sender, json, *args, **kwargs)
     populate_title_suggest(sender, json, *args, **kwargs)
 
 
+def populate_author_suggest(sender, json, *args, **kwargs):
+    """Populate the ``author_suggest`` field of Authors records."""
+    if 'authors.json' not in json.get('$schema'):
+        return
+
+    author_paths = [
```

Comment:
@michamos we will leave it like this, ready for the new schema and merge it when the schema is stable and through. Looks ok?